recipe/meta.yaml

```
@@ -163,18 +198,20 @@ outputs:
 
   - name: gql-with-all
     build:
-      noarch: python
     requirements:
       host:
-        - {{ min_python }}
+        - python
         - pip
+        - wheel
+        - setuptools
       run:
         - {{ pin_subpackage('gql-with-aiohttp', exact=True) }}
+        - {{ pin_subpackage('gql-with-httpx', exact=True) }}
         - {{ pin_subpackage('gql-with-botocore', exact=True) }}
         - {{ pin_subpackage('gql-with-requests', exact=True) }}
         - {{ pin_subpackage('gql-with-websockets', exact=True) }}
         - {{ pin_subpackage('gql', exact=True) }}
-        - {{ min_python }}
+        - python
     test:
```

```suggestion
    test:
      - gql
      - gql.transport.httpx
      - gql.transport.aiohttp
      - gql.transport.requests
      - gql.transport.appsync_auth
      - gql.transport.websockets
```

This should satisfy the linter, and a bit more testing never hurts.